### ISSUE TITLE
Improve accessibility of superscript symbols & footer

### ISSUE THREAD

**Issue by aquilent-dgemoets** [USDepartmentofLabor/Child-Labor#56]
The Legal Standards and Enforcement screens use symbols to denote things like "age calculated based on available information," "does not meet international standards," and the like.  There are symbols (currently asterisk, double-dagger, and phi) that refer to a footer where the symbols are explained.

For better accessibility, the symbol explanation should simply be spoken along with the value, e.g., "Minimum Age for Hazardous Work, Yes, 18, age calculated based on available information."

Furthermore, ideally the footer should not be spoken.  At a minimum, there should be a period at the end of each symbol explanation so that there is a pause between each explanation.

**Comment by aquilent-dgemoets** [comment]
This is addressed in the latest beta (9/9/16), but not to our complete satisfaction.

The symbol explanation is spoken along with the value as described above, but due to an apparent limitation in the iOS accessibility platform, we cannot tell VoiceOver NOT to speak the footer.

If there's time before the final release, we will try to revisit this issue and find a solution to get VoiceOver not to speak the footer.

**Comment by jfc3-dol** [comment]
Looks like the special symbols have been removed from that random countries and pages I checked so going to close the issue.
